### PR TITLE
Rename nteract-scrapbook to scrapbook

### LINKED ISSUE #15289: reporting deprecated warnings saw in breeze
when working with breeze I saw these warnings many times in the logs:

 ```
 =============================== warnings summary ===============================
  tests/providers/amazon/aws/log/test_cloudwatch_task_handler.py::TestCloudwatchTaskHandler::test_close_prevents_duplicate_calls
    /usr/local/lib/python3.6/site-packages/jose/backends/cryptography_backend.py:18: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
      from cryptography.utils import int_from_bytes, int_to_bytes
      
 tests/always/test_example_dags.py::TestExampleDags::test_should_be_importable
    /usr/local/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
      return f(*args, **kwds)
  
  tests/always/test_example_dags.py::TestExampleDags::test_should_be_importable
    /usr/local/lib/python3.6/site-packages/scrapbook/__init__.py:8: FutureWarning: 'nteract-scrapbook' package has been renamed to `scrapbook`. No new releases are going out for this old package name.
      warnings.warn("'nteract-scrapbook' package has been renamed to `scrapbook`. No new releases are going out for this old package name.", FutureWarning)
```

### PR DESCRIPTION
**Pull request by xinbinhuang**
close: #15289
The package is renamed `nteract-scrapbook` to `scrapbook`. The old package will not be updated for new releases.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -384,8 +384,8 @@ def get_sphinx_theme_version() -> str:
     'pdpyras>=4.1.2,<5',
 ]
 papermill = [
-    'nteract-scrapbook[all]>=0.3.1',
     'papermill[all]>=1.2.1',
+    'scrapbook[all]>=0.3.1',
```
Comment (eladkal) [inline setup.py]: Why do we need to pin min version?
`scrapbook` starts from version `0.5.0`

https://pypi.org/project/scrapbook/#history
https://pypi.org/project/nteract-scrapbook/#history

**Review by eladkal — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (xinbinhuang) [reply setup.py]: Good point!

**Comment by xinbinhuang** [comment]
@eladkal  updated!

File: setup.py [hunk 2]
```diff
@@ -384,8 +384,8 @@ def get_sphinx_theme_version() -> str:
     'pdpyras>=4.1.2,<5',
 ]
 papermill = [
-    'nteract-scrapbook[all]>=0.3.1',
     'papermill[all]>=1.2.1',
+    'scrapbook[all]>=0.5.0',
```
Comment (eladkal) [inline setup.py]:
```suggestion
    'scrapbook[all]',
```
Since its already the minimum version

**Review by eladkal — APPROVED** [review]
LGTM
Left one comment.

**Pull request merged**